doofinder-for-woocommerce/includes/api/class-doofinder-api.php

```diff
@@ -509,6 +405,91 @@ public function send_batch( $items_type, array $items, $language = null ) {
 		$this->log->log('End Send Batch' . "\n");
 	}
 
+	public function create_temporary_index($items_type){

```

Comment:
Nice job!!!
Perhaps to polish it further it could be subdivided into other functions so as not to leave a giant monolith?

Reply:
Hahaha, this project is built with tons of monoliths but, good idea, I'll start breaking this one!
Thanks!